.github/workflows/build_publish_openmrs.yml

```diff
@@ -69,50 +49,36 @@ jobs:
           file: package/docker/openmrs/Dockerfile
           push: true
           tags: bahmni/openmrs:${{ env.ARTIFACT_VERSION }},bahmni/openmrs:latest
+      - name: Update Version and Image Tag
+        run: |
+          yq --inplace '.image.tag = "${{ env.ARTIFACT_VERSION }}"' $HELM_CHART_PATH/values.yaml
+          yq --inplace '.version = "${{ env.ARTIFACT_VERSION }}"' $HELM_CHART_PATH/Chart.yaml
```

Comment:
since we are squashing docker and helm in same job - consider changing the job name